api/types.go

```diff
@@ -10,12 +10,17 @@ import (
 )
 
 type Incident struct {
+	ID     uuid.UUID `json:"id"`
+	Title  string    `json:"title"`
+	TeamID uuid.UUID `json:"team_id"`
+	Team   Team      `json:"team"`
```

Comment:
Why both ?

Reply:
I initially thought there maybe usecases where we may need teamID but not the entire team object but I don't think thats the case for now, have removed it from here and other places as well